products/caasp/main.pm

```diff
@@ -128,6 +129,8 @@ sub load_feature_tests {
         if (is_caasp('kubic') && check_var('SYSTEM_ROLE', 'plain')) {
             loadtest 'console/skopeo';
             loadtest 'console/umoci';
+            loadtest 'console/runc';
+            loadtest 'console/rootless';
```

Comment:
The answer to this can be split in two parts:

**Package should be present:**

I've checked via https://scc.suse.com/packages

* runc: SLE-Module-Containers12-Updates (s390x, x86_64, ppc64le), SUSE CaaSP 2.0, 3.0, 12-SP3, 12-SP4 (?), TW, Leap

* skopeo: SLE15 Basesystem Module (aarch64,s390x, x86_64, ppc64le), TW, Leap

* umoci: TW, Leap

* rootless: this can run only when all of the above three packages are available, e.g. TW, Leap


**Modification in the code:**

In that case, I have to modify the code to run 'zypper install' instead of 'transtactional_update' when it comes to SLEx.

Reply:
Well, are you willing to do that? If not then please create a ticket and we will handle it ... eventually. If you don't want to create a ticket I will ... Unless I forget ;)